rectangles.js

```diff
@@ -0,0 +1,24 @@
+var rectangle = {width: 10, height: 20}
+
+function area(rectangle) {
+  return rectangle.width * rectangle.height
+}
+
+function perimeter(rectangle) {
+  return (rectangle.width * 2) + (rectangle.height * 2)
+}
+
+function diagonal(rectangle) {
+  var add = (Math.pow(rectangle.width, 2)) + (Math.pow(rectangle.height, 2))
+  return (Math.sqrt(add))
+}
+
+function isSquare(rectangle) {
+  if (rectangle.width === rectangle.height) {
+    return true
+  } else {
+    return false
+  }
+}
+
+module.exports = {area: area, perimeter: perimeter, diagonal: diagonal, isSquare: isSquare}
```

Comment:
Protip: you should be able to just `module.exports = {area, perimeter, diagonal, isSquare}` because you use the same name for key and value